### ISSUE TITLE
Plugin cuts out spaces in the start of lines for JSON

### ISSUE THREAD

**Issue by wcoder** [wcoder/highlightjs-line-numbers.js#39]
**Describe the bug**
I see the problem with spaces in the start of lines. Plugin cuts them out. From #38 

**To Reproduce**
Example for reproduce:
```json
{
  "averageBuyFee" : 1.3999999999999999,
  "timestamp" : 1527721200000,
  "rates" : {
    "HUF" : 274.05874999999997,
    "HRK" : 6.3338999999999999,
    "CDF" : 1614.661462,
    "SDG" : 18.044215000000001,
    "CZK" : 22.1663,
    "BAM" : 1.6809190000000001
  },
  "base" : "USD",
  "excludedCurrencies" : [
    "XDR",
    "ZMK",
    "BTC"
  ]
}
```

**Expected behavior**
Spaces don't be cut out.

**Screenshots**
![image](https://user-images.githubusercontent.com/766193/40904242-c2040d0c-67e2-11e8-9bbc-739d9867c60e.png)

**Desktop (please complete the following information):**
 - All

**Smartphone (please complete the following information):**
 - All

**Comment by klische** [comment]
Would you happen to have an ETA for when this issue would be fixed?

**Comment by wcoder** [comment]
Hi @klische 

I will try to fix it as soon as possible.

### Workaround

You can use previous major version `1.1.0`:

Source:
https://cdnjs.cloudflare.com/ajax/libs/highlightjs-line-numbers.js/1.1.0/highlightjs-line-numbers.min.js

Documentation:
https://github.com/wcoder/highlightjs-line-numbers.js/blob/v1.1.0/README.md#usage

**Comment by wcoder** [comment]
Has been fixed in v2.6
